src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js

```diff
@@ -368,15 +372,16 @@ export function SingleLineDiagramPane({
                 } else {
                     updateSld();
                 }
-            } else if (
+            } // for what is used this else bloc ?
```

Comment:
For me, ok to remove this block. When a node is buit, the SLD is updated via the nodeUpdated modification

Reply:
yes,  remove it if it's not used

Reply:
done, i removed